packages/modeling/src/geometries/geom2/toSides.js

```diff
@@ -1,8 +1,7 @@
 const applyTransforms = require('./applyTransforms')
 
 /**
- * Produces an array of sides from the given geometry.
- * The returned array should not be modified as the data is shared with the geometry.
+ * Produces a new array of sides from the given geometry.
```

Comment:
This is not true if the transforms are 'identity'. So the initial comment is correct, sometimes.

Reply:
changed the comment a bit to be more precise :)

Reply:
And sides should never be modified....